corehq/motech/openmrs/tasks.py

```diff
@@ -188,13 +189,9 @@ def import_patients_to_domain(domain_name, force=False):
     :param domain_name: The name of the domain
     :param force: Import regardless of the configured import frequency / today's date
     """
-    today = datetime.today()
     for importer in get_openmrs_importers_by_domain(domain_name):
-        if not force and importer.import_frequency == IMPORT_FREQUENCY_WEEKLY and today.weekday() != 1:
-            continue  # Import on Tuesdays
-        if not force and importer.import_frequency == IMPORT_FREQUENCY_MONTHLY and today.day != 1:
-            continue  # Import on the first of the month
-        # TODO: ^^^ Make those configurable
+        if not _should_import_today(importer) and not force:
```

Comment:
can this be called more than once per day?

Reply:
It is scheduled to be called daily at 04h04 ... but a user can call it as many times as they want by using an "Import Now" button on the page where Importers are configured. (If they do that, `force` is `True`.)